common-arcana.lic

```diff
@@ -406,6 +406,8 @@ module DRCA
       .spell_data
       .select { |_name, properties| properties['cyclic'] }
       .select { |name, _properties| DRSpells.active_spells.keys.include?(name) }
+      .reject { |_name, properties| properties['skip_release'] }
+      .reject { |_name, properties| properties['harmless'] }
```

Comment:
💡 The list of spells being filtered from come `get_data('spells')` on line 405, which does not have any per-character customizations in them. For example, if `skip_release` is a property people configure in their yaml then that property won't be set here. On the other hand, `harmless` is defined in `base-spells.yaml` so it will be useful as a filter here.